src/qofonomanager.cpp

```diff
@@ -106,6 +106,7 @@ void QOfonoManager::onModemRemoved(const QDBusObjectPath& path)
 void QOfonoManager::onGetModemsFinished(QDBusPendingCallWatcher* watcher)
 {
     QDBusPendingReply<ObjectPathPropertiesList> reply(*watcher);
+    watcher->deleteLater();
```

Comment:
I don't understand how this changes anything.  The watcher won't be deleted unless control returns to the event loop, and I don't see anything between here and line 127 that looks like it will affect the event loop.